iati/core/constants.py

```diff
@@ -20,6 +21,11 @@
 STANDARD_VERSION_LATEST = max(STANDARD_VERSIONS)
 """The latest version of the IATI Standard."""
 
+STANDARD_VERSIONS_MAJOR = list(set([
+    int(math.floor(float(v))) for v in STANDARD_VERSIONS
```

Comment:
Using `math` operations rather than string manipulation (`split('.')[0]` sort of thing) is unable to work with SemVer.

Reply:
Given that we don't use SemVer [yet](https://discuss.iatistandard.org/t/revision-of-iati-standard-upgrade-procedures/1014/4), I would suggest this is rectified at a later point.

Reply:
Since they're strings in the XML (well, at V2 at least - they're decimals at V1!), it just seems a bit odd doing math operations on them.